### ISSUE TITLE
Key error while reading class labels

### ISSUE THREAD

**Issue by shravanambudkar** [HyoungsungKim/Relation_network_for_term_project#2]
Hey! I am running this code in python 3.6 and pytorch 1.7 
while running the last part of test_main.ipynb i am running into this error
KeyError: '..\\datas\\omniglot_resized'

can someone pls help me

**Comment by HyoungsungKim** [comment]
Please let me know more detail

**Comment by shravanambudkar** [comment]
Yes, @HyoungsungKim  I completed the following steps,
1. Cloned the repository
2. Installed pytorch 1.7 with anaconda environment python 3.6
3. Installed other requirements.
4. Unzipped the omniglot resized images in the datas folder.
3. Run the test_main.ipynb file to check your model results.

Ran into the following error:

KeyError: '..\datas\omniglot_resized'

in the line 85 of the task_generator_no_args.py file.
I guess it should point out to the class label but it is pointing out the data directory.
This is the screenshot of the error,

![image](https://user-images.githubusercontent.com/61011836/100702648-15f84500-33c8-11eb-968d-ccdbffa1ee66.png)

Can you help please!

**Comment by HyoungsungKim** [comment]
![Screenshot_20201201_152118](https://user-images.githubusercontent.com/25428581/100704453-77ef8500-339d-11eb-8be7-a7a460205e67.png)

Thank you for visiting my repository and making a issue.

 I tested by following the step you said but it works well for me.

I recommend restart ipynb kernel and run from first kernal again.

If you trained well by using `train_main.ipynb`, then `task_generator_no_args.py` is not a reason of problem.

**Comment by shravanambudkar** [comment]
Yes, I tried restarting the kernel, also for training it is giving me the same issue,
I even tried implementing it on another pc installing everything from scratch,
I am running it on windows 10, 
can it be os specific error?

**Comment by HyoungsungKim** [comment]
I tested on Linux but i think it may not a OS sepcific error.

How did you locate `omniglot_resized`?

It has to be located in `datas` folder 

Namley,

`datas ` --- `omniglot_resized`

**Comment by shravanambudkar** [comment]
Yes I have extracted the zip file there itself
![image](https://user-images.githubusercontent.com/61011836/100711924-f9fc9f80-33d7-11eb-830a-813c1ab9cfb2.png)

Inside datas --- omniglot_resized
